### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rhiannonlynne**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rhiannonlynne** [comment]
@conda-forge-admin, please rerender

**Review by ehneilsen — APPROVED** [review]
I suspect that at least some of the requirements removed (healsparse, selenium) may need to get added to the test imports section? I'm not sure how the test section here gets used.

**Comment by rhiannonlynne** [comment]
The test section here just checks that schedview will import as it is installed with the dependencies listed. I think that's all we need to do here, as we run the unit tests in the schedview repo directly. I don't think you add the additional "test requirements" dependencies here, because someone installing from conda would not be running the unit tests themselves. 
However, it's likely that we would want to add heal-sparse and selenium to the test-requirements.txt file in the schedview repo (and any other dependencies that need to be present to run the unit tests) .. I think we currently have them in the requirements.txt file? This may be fine as well. The requirements in the requirements.txt file are intended to be "for a developer" so it also seems appropriate for them to be there. 
In short, I don't think we need to do anything else really.
